ansible/tasks/static-sticky.yml

```diff
@@ -0,0 +1,99 @@
+---
+
+- name: "Add user static-sticky"
+  user:
+    name: "static-sticky"
+    shell: "/usr/sbin/nologin"
+    home: "/var/www/static-sticky"
+    system: true
+
+- name: "copy systemd unit"
+  template:
+    src: "templates{{ item }}.j2"
+    dest: "{{ item }}"
+  with_items:
+    - "/etc/systemd/system/deploy-static-sticky.service"
+
+- name: "ensure polkit rules dir exists"
+  file:
+    path: "/etc/polit-1/rules.d"
+    state: "directory"
+
+- name: "copy polkit rules"
+  template:
+    src: "templates{{ item }}.j2"
+    dest: "{{ item }}"
+  with_items:
+    - "/etc/polkit-1/rules.d/00-deploy-static-sticky-service.rules"
+
+- block:
+  - name: "copy deploy script"
+    template:
+      src: "templates{{ item }}.j2"
+      dest: "{{ item }}"
+      mode: "u+rwx"
+    with_items:
+      - "/var/www/static-sticky/deploy_static_sticky.sh"
+
+  - name:
+      "clone static-sticky repository"
+    git:
+      repo: "https://github.com/svsticky/static-sticky.git"
+      dest: "/var/www/static-sticky/static-sticky"
+      version: "master"
+      depth: 1
+      force: true
+    diff: false
+
+  - name:
+      "template environment file for static-sticky"
+    template:
+      src: "templates{{ item }}.j2"
+      dest: "{{ item }}"
+    diff: false
+    with_items:
+      - "/var/www/static-sticky/static-sticky/.env"
+
+  - name: "clone NVM"
+    git:
+      repo: "https://github.com/creationix/nvm.git"
+      dest: "/var/www/static-sticky/.nvm"
+      depth: 1
+    diff: false
+
+  - name: "install node version"
+    shell: |
+      . /var/www/static-sticky/.nvm/nvm.sh
+      nvm install
+    args:
+      chdir: "/var/www/static-sticky/static-sticky"
+      executable: "/bin/bash"
+
+  - name: "run npm install"
+    shell: |
+      . /var/www/static-sticky/.nvm/nvm.sh
+      npm install
+    args:
+      chdir: "/var/www/static-sticky/static-sticky"
+      executable: "/bin/bash"
+
+  - name: "build website"
+    shell: |
+      . /var/www/static-sticky/.nvm/nvm.sh
+      npm run build
+    args:
+      chdir: "/var/www/static-sticky/static-sticky"
+      executable: "/bin/bash"
+
+  - name: "copy website contents to webroot"
+    synchronize:
+      src: "/var/www/static-sticky/static-sticky/public/"
+      dest: "/var/www/static-sticky/alpha.{{ canonical_hostname }}/"
+      delete: true
+      archive: true
+
+    # NOTE: The trailing slashes are significant!
```

Comment:
Can you move this comment to a bit higher in the file? This might be easy to miss